### PR TITLE
Feature/alert if license is missing

### PR DESCRIPTION
**Pull request by mplieske**
fixed:
- licenses are shown in linux (tested on Ubuntu 18.04 LTS)

### CODE REVIEW

File: src/main/java/de/doubleslash/keeptime/common/FileOpenHelper.java [hunk 1]
```diff
@@ -56,12 +56,14 @@ private static void executeCommandWindows(final Runtime rt, final File file) {
       }
    }
 
-   private static void executeCommandLinux(final Runtime rt, final File file) {
+   private static void executeCommandLinux(final Runtime rt, final String fileString) {
       try {
-         final String command = "xdg-open " + file;
-         LOG.debug("executing command: {}", command);
+         final String command = fileString;
+         LOG.debug("executing command: gedit {}", command);
```
Comment (Death111) [inline src/main/java/de/doubleslash/keeptime/common/FileOpenHelper.java]: gedit?

File: src/main/java/de/doubleslash/keeptime/common/BrowserHelper.java [hunk 2]
```diff
@@ -50,10 +50,12 @@ private static void executeCommandWindows(final Runtime rt, final String url) {
 
    private static void executeCommandLinux(final Runtime rt, final String url) {
       try {
-         final String command = "xdg-open " + url;
-         LOG.debug("execute command: {}", command);
+         final String command = url;
+         LOG.debug("execute command: xdg-open {}", command);
```
Comment (Death111) [inline src/main/java/de/doubleslash/keeptime/common/BrowserHelper.java]: macht mehr sinn das string [] auszugeben :)

File: src/main/java/de/doubleslash/keeptime/common/FileOpenHelper.java (hunk 1 above)
Comment (Death111) [reply src/main/java/de/doubleslash/keeptime/common/FileOpenHelper.java]: see other comment with printing the []

**Comment by Death111** [comment]
When you tested it again on linux after the last changes @mplieske let me know. then we can merge it.

**Comment by mplieske** [comment]
tested, corrected and working on Ubuntu 18.04.1 LTS :)

**Pull request merged**